knative-operator/hack/update-deps.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+readonly ROOT_DIR=$(dirname $0)/..
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# Prune modules.
+go mod tidy
+go mod vendor
+
+# TODO: Remove this once we bump kubernetes versions.
+git apply "$ROOT_DIR/hack/manifestival.patch"
+
+rm -rf $(find vendor/ -name 'OWNERS')
+# Remove unit tests & e2e tests.
+rm -rf $(find vendor/ -path '*/pkg/*_test.go')
+rm -rf $(find vendor/ -path '*/e2e/*_test.go')
+
```

Comment:
Upstream script also removes LICENSE files. 

Other remarks:

* You could combine to a single find 
* This command will break totally if files names contains spaces. So user `xargs` instead.
* It also breaks if there are too many files so that the overall command line becomes too long.
* Why not deleting just all `*_test.go` files ?

Reply:
Recommended form to fix those issues:

```
find "vendor/" \( -name "OWNERS" -o -name "*_test.go" \) -print0 | xargs -0 rm -f
```

Reply:
Totally copied from upstream, lemme check.